content/3_Silver/Func_Graphs.mdx

```diff
@@ -390,66 +396,84 @@ import java.util.*;
 
 public class badge
 {
-	static InputReader in = new InputReader(System.in);
-	static PrintWriter out = new PrintWriter(System.out);
+	static BufferedReader reader;
+	static PrintWriter writer;
 	public static final int MN = 1010;
 	public static int N;
```

Comment:
why is 'N'a global variable? Should be fine to leave it as local since you don't use it anywhere else.